packages/wrangler/src/tail.tsx

```diff
@@ -8,6 +8,61 @@ export type TailApiResponse = {
   expires_at: Date;
 };
 
+export type CliFilters = {
```

Comment:
include the word `Tail` otherwise it's not clear inside other contexts what it's used for, and CLI should be capitalised 
```suggestion
export type TailCLIFilters = {
```

Reply:
i must kill the rust self to embrace the js self...from https://rust-lang.github.io/api-guidelines/naming.html:

> In UpperCamelCase, acronyms and contractions of compound words count as one word: use `Uuid` rather than `UUID`, `Usize` rather than `USize` or `Stdin` rather than `StdIn`.